packages/rocketchat-slashcommands-join/server.js

```diff
@@ -0,0 +1,39 @@
+
+/*
+* Join is a named function that will replace /join commands
+* @param {Object} message - The message object
+*/
+
+
+const Join = function(command, params, item) {
```

Comment:
IMHO named functions are better for theses cases, cuz the stack trace of the errors will be more clear.
What do you think about change `const Join = function(command...` by `function Join(command...`?